dbfit-java/core/src/main/java/dbfit/api/PreparedDbCommand.java

```diff
@@ -0,0 +1,18 @@
+package dbfit.api;
+
+import java.sql.SQLException;
+
+/**
+ * Represents a prepared database statement with ability to bind input parameters and
+ * to register and access output ones including return values.
+ */
+public interface PreparedDbCommand extends DbCommand, DbQuery {
```

Comment:
We don't tie to JDBC details - just borrowing the same convention. I think it's worth considering `ParameterisedDbCommand`. It raises a question about `getGeneratedKey` - as it doesn't seem to be related to the parameterization.